models/files.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var files = sequelize.define('files', {
```

Comment:
@cameronwolf this is sequelize code setup, will have to check if that breaks anything.